FlipperZero/.swiftlint.yml

```diff
@@ -5,11 +5,81 @@ excluded:
  - Frameworks
 
 disabled_rules:
- - force_try
  - identifier_name
- - line_length
- - force_cast
- - trailing_whitespace
```

Comment:
`trailing_whitespace` is annoying when using `⌃i` to re-indent selection. 
Auto-indented newlines would have 4 spaces indentation which produces a warning with this rule.

Reply:
I think this Xcode setting should help with it
![XcodeTrimWhitespaces](https://user-images.githubusercontent.com/10916937/95944539-41ce5580-0d9d-11eb-895a-c8147d506ce0.png)